src/App.jsx

```diff
@@ -73,15 +79,18 @@ const App = () => {
      * @function fetchData
      */
     async function fetchData() {
-      await generateUsersList(session);
```

Comment:
This function is needed to create our usersList. Should be kept. Perhaps refactor generateUserList instead with the getPodUrlAll function inside it.

Reply:
This functionality has been moved to `loadUserList` in `model-helpers/UserList.js`